### PR TITLE
Don't fail on python 3, give better message

### PR DESCRIPTION
**Pull request by danielfrg**
@mrocklin this is how I would solve the python 3 click "issue".

### CODE REVIEW

File: distributed/cli/utils.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+py3_err_msg = """You are using Python 3 and your system is not compatible with the Unicode text model in Python 3
+This usually gets solved by exporting this environment variables but it depends on your system:
+export LC_ALL=C.UTF-8
+export LANG=C.UTF-8
+
+For more information see: http://click.pocoo.org/5/python3/
+"""
```
Comment (mrocklin) [inline distributed/cli/utils.py]: This looks good to me.  Can we reverse the order of the language to say first that the terminal doesn't support unicode and second that this is an issue for Python 3?

Comment (danielfrg) [inline distributed/cli/utils.py]: Feel free to change the message like you want.

Comment (danielfrg) [inline distributed/cli/utils.py]: Or tell me the message you want.

Comment (mrocklin) [inline distributed/cli/utils.py]: OK, I'll merge this, edit, and push manually

**Comment by mrocklin** [comment]
Is this accurate?  https://github.com/dask/distributed/commit/177dd5eae5bd4a98b6bd25be22650f23de66d1eb

**Comment by danielfrg** [comment]
LGTM

**Pull request merged**